pkg/pmem-csi-operator/controller/deployment/controller_driver.go

```diff
@@ -37,6 +43,27 @@ const (
 	nodeControllerPort    = 10001
 )
 
+// A list of all object types potentially created by the
+// operator. It's okay and desirable to list more than actually used
+// at the moment, to catch new objects.
+// NOTE: Add the new object types to this list.
+// But, This list should not shrink even if the later versions of
+// operator does not install the object listed here.
+// map: GVK -> isClusterScoped
+var allObjectTypes = map[schema.GroupVersionKind]bool{
```

Comment:
This is similar the list that we have in test/e2e/operator/validate/validate.go.

Please refactor so that we have only one list.

Also add a comment that we need to keep this in sync with RBAC rules.